bpf/granted.c

```diff
@@ -111,7 +111,7 @@ granted_pkt(struct gk_bpf_pkt_ctx *ctx)
 		priority = PRIORITY_RENEW_CAP;
 	}
 
-	if (gk_bpf_encapsulate(ctx, priority) < 0)
+	if (gk_bpf_encapsulate(ctx, priority, ctx->direct_if_possible) < 0)
```

Comment:
Replace `ctx->direct_if_possible` with `false`. We are keeping the granted program 100% compatible with the granted state builtin in Gatekeeper. Future programs will pass `true` here.

Reply:
We'll need `#include <stdbool.h>` to use `true` and `false`.